webvtt.go

```diff
@@ -59,9 +59,23 @@ func ReadFromWebVTT(i io.Reader) (o *Subtitles, err error) {
 	var item = &Item{}
 	var blockName string
 	var comments []string
+
+	// For the Index Counter
+	saveIndex := true
+	var index int
+	var convErr error
+
 	for scanner.Scan() {
 		// Fetch line
 		line = strings.TrimSpace(scanner.Text())
+
+		// Fetch Index
```

Comment:
I'd rather not extract the index this way. I'd rather parse it [here](https://github.com/asticode/go-astisub/blob/master/webvtt.go#L188)

Reply:
First off, there's a bug in the library : can you remove the extra space after `STYLE` [here](https://github.com/asticode/go-astisub/blob/master/webvtt.go#L109) ?

Second, when I add `fmt.Println(line)` [here](https://github.com/asticode/go-astisub/blob/master/webvtt.go#L188), I do get this with the example file :

```
1
2
4
5
6
```

Therefore I do get all indexes, and you should be able to parse them.

Reply:
The bug has been fixed now with https://github.com/asticode/go-astisub/pull/22/commits/4adf45bbc0604cf1200febf067bd524d1cb1e451. All this time, I was looking "example-out.vtt" file. But now, logic is changed to save indexes to the corresponding Items of the Subtitles [Items which doesn't have an index/identifier will be populated with 0]. that commit is in https://github.com/asticode/go-astisub/pull/22/commits/3447ccf31a7bd383610d60cce1c59ff35dbe70ee.